device_specific/flash_the_ttgo_tdisplay

```diff
@@ -81,25 +93,183 @@ fi
 cd "${jade_save_directory}"
 jade_version="$(git describe --tags)"
 
+pip install cbor2
+export PYTHONPATH="$jade_save_directory"
+
+secure_boot_status=$(
+  python << EOF
+from jadepy import JadeAPI
+import _cbor2
+import serial
+
+create_jade_fn = JadeAPI.create_serial
+kwargs = {'device': None, 'timeout': 5}
+
+try:
+    with create_jade_fn(**kwargs) as jade:
+        verinfo = jade.get_version_info()
+
+    if 'SB' in verinfo['JADE_FEATURES']:
+        print('enabled')
+    else:
+        print('disabled')
+except _cbor2.CBORDecodeEOF:
+    print('uninitialized')
+except serial.serialutil.SerialException:
+    print('devicenotfound')
+EOF
```

Comment:
Should we put this script in a separate `.py` file?

Reply:
I like to keep redundancies small but in this case i see problems if we move this to a separate file. We would then need to pull it in every script with curl from a fixed url. This has major disadvantages:

- You execute code unknown code from an http endpoint
- Pulling stuff from online resources prevents offline usage
- You cant test different python code in separate git branches because every time you run the script it will be pulled from master (i see this problem currently with the dependency pulling)